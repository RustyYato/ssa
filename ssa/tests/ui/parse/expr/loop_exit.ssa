let x = loop {
    break continue return break;
};
