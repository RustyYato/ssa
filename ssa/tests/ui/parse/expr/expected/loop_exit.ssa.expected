[
    id: 1
    kind:
        Let:
            binding:
                id: 1
                name: "x"
            ty: null
            value:
                id: 5
                kind:
                    Loop:
                        block:
                            stmts: [
                                id: 1
                                kind:
                                    Expr:
                                        id: 4
                                        kind:
                                            Break:
                                                id: 3
                                                kind:
                                                    Continue:
                                                        id: 2
                                                        kind:
                                                            Return:
                                                                id: 1
                                                                kind:
                                                                    Break: null
                            ]
                            expr: null
]